### PR TITLE
Quincy NFT command

### PR DESCRIPTION
**Pull request by Landmaster**
Command syntax: `/nft unlock_round start_round end_round difficulty [better_sell_deals]`

### CODE REVIEW

File: slash_commands/nft.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { red, cyber } = require('../jsons/colors.json');
+const bHelper = require("../helpers/bloons-general.js");
+
+const builder = new SlashCommandBuilder()
+.setName('nft')
+.setDescription('Calculate Quincy Action Figure buy/sell price in Shol Geraldo\'s shop')
+.addIntegerOption(option => option.setName('unlock_round').setDescription("Start of round when the action figure is unlocked").setRequired(true).setMinValue(1))
```
Comment (rmlgaming) [inline slash_commands/nft.js:8]: Would it be more straightforward to say "The round Geraldo is placed"?

File: slash_commands/nft.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        { name: 'Medium (Military only, Reverse, Apopalypse)', value: 'medium' },
+        { name: 'Hard (Magic only, Double HP)', value: 'hard' },
+        { name: 'Impoppable', value: 'impoppable' }
+    )
+)
+.addBooleanOption(option => option
+    .setName('better_sell_deals')
+    .setDescription("Whether the Better Sell Deals MK is enabled")
+    .setRequired(false)
+);
```
Comment (rmlgaming) [inline slash_commands/nft.js:26]: I wonder if this should be default on? Also I think having an optional boolean option is kinda redundant because one of the values is going to be the same as if you skipped the argument (`better_sell_deals: false` in this case)

It's up to you but maybe you could call this "turn_off_mk" with one string option of "yes". The description can be "Whether to disable the Better Sell Deals MK".

**Review by rmlgaming — COMMENTED** [review]
Nice!

File: slash_commands/nft.js (hunk 1 above)
Comment (Landmaster) [reply slash_commands/nft.js:8]: I believe that that's not necessarily the same thing if MK is not on

Comment (rmlgaming) [reply slash_commands/nft.js:8]: Ah ok

**Comment by rmlgaming** [comment]
@Landmaster just let @hemisemidemipresent know if this is good to go. This is good enough for me!

**Review by rmlgaming — APPROVED** [review]

**Review by rmlgaming — APPROVED** [review]

**Comment by Landmaster** [comment]
@hemisemidemipresent command should be gtg, maybe test some edge case rounding first though
